### PR TITLE
now using dynamic keys for theme objects

### PR DESCRIPTION
**Pull request by klazbaba**
This PR fixes themes not working.
Adding for example stylesheet.calendar.main to theme is not picked up by the styles defined in the component because the object key contains a ".", which is not valid in a variable name.

### CODE REVIEW

File: src/calendar/day/marking/style.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default function styleConstructor(theme: Theme = {}) {
       borderBottomRightRadius: 2,
       marginRight: 4
     },
-    ...(theme.stylesheet?.marking || {})
+    //@ts-ignore
+    ...(theme['stylesheet.marking '] || {})
```
Comment (Klemminn) [inline src/calendar/day/marking/style.ts:30]: Probably shouldn't be a space at the end of 'stylesheet.marking '?

Comment (klazbaba) [reply src/calendar/day/marking/style.ts]: True, thanks for pointing that out.

**Review by Klemminn — APPROVED** [review]

**Comment by hraschan** [comment]
Can someone approve those changes? Would be really helpful! :)

**Review by hraschan — APPROVED** [review]
Everything looks good to me.

**Comment by theneekz** [comment]
This bug applies to all of the theme ids, not just markings:
"stylesheet.calendar.header"
"stylesheet.calendar.main"
"stylesheet.calendar-list.main"
"stylesheet.agenda.main"
"stylesheet.agenda.list"
"stylesheet.day.basic"
"stylesheet.day.single"
"stylesheet.day.multiDot"
"stylesheet.day.period"
"stylesheet.dot"

**Comment by Klemminn** [comment]
@Inbal-Tish could you please take a look? :)
This feature is working in older releases, would be wonderful to get it back.

**Comment by Klemminn** [comment]
@ethanshar @emilisb this seems like a minor but needed change.... can either of you take a look? :)

**Review by Inbal-Tish — APPROVED** [review]

**Comment by theneekz** [comment]
:) 👍

**Pull request merged**